tailwind.config.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
           '30a': '#EAEBED',
           '40': '#DDDDDD',
           '50a': '#CFD1D5',
+          'blue': '#EBF2FC',
+          'subFooter': '#F5F5F5',
```

Comment:
this is a gray but very light so maybe a number like '10' or 'lightest'? instead of subfooter as it doesn't describe the colour in any way.